jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/ProcessServiceImplTest.java

```diff
@@ -169,6 +170,25 @@ public void testStartProcessWithParms() {
     	assertNull(pi);
     }
 
+    @Test
+    public void testStartSynchronousProcessWithParms() {
+        assertNotNull(deploymentService);
+
+        KModuleDeploymentUnit deploymentUnit = new KModuleDeploymentUnit(GROUP_ID, ARTIFACT_ID, VERSION);
+
+        deploymentService.deploy(deploymentUnit);
+        units.add(deploymentUnit);
+        assertNotNull(processService);
+
+        Map<String, Object> params = new HashMap<String, Object>();
+        params.put("name", "test");
```

Comment:
This line is not really needed, as the final variable value is set in the inner script

Reply:
ok.